docs/articles/expensify-classic/bank-accounts-and-credit-cards/company-cards/Connect-Company-Cards.md

```diff
@@ -1,5 +1,91 @@
+
+# How to export company card expenses to a connected accounting integration
+
+## Pooled GL account 
+
+To export credit card expenses to a pooled GL account:
+Go to Settings > Policies > Group > Policy Name > Connections > Accounting Integrations > Configure
+Select Credit Card / Charge Card / Bank Transaction as your Non-reimbursable export option. 
+Please review the Export Settings page for exporting Expense Reports to NetSuite
+Select the Vendor/liability account you want to export all non-reimbursable expenses to.
+
+## Individual GL account 
+
+Go to Settings > Domain > Domain name > Company Cards
+Click the Export Settings cog to the right-hand side of the card and select the GL account you want to export expenses to. 
+You're all done! After the account is set, exported expenses will be mapped to the specific account selected.
+
+# Deep Dive
+## Identifying company card transactions
+When you link your credit cards to Expensify, the transactions will appear in each user's account on the Expenses page as soon as they're posted. You can identify transactions from centrally managed cards by seeing the locked card icon next to them. That icon indicates that they’re company card expenses:
+[add image here]
+
+## Importing historical transactions 
+
+After a card is connected via direct connection or via Approved! banks, Expensify will import 30-90 days' worth of historical transactions to your account (the timeframe is based on your bank's discretion). Any historical expenses beyond that date range can be imported using the CSV spreadsheet import method. 
+
+## Using eReceipts 
+Expensify eReceipts serve as digital substitutes for paper receipts in your purchase transactions, eliminating the necessity to retain physical receipts or utilize SmartScanning receipts. In the case of Expensify Card transactions, eReceipts are automatically generated for all amounts. For other card programs, eReceipts are specifically generated for purchases amounting to $75 or less, provided the transactions are in USD.
+To ensure seamless automatic importation, it's essential to maintain your transactions in US Dollars. Additionally, eReceipts can be directly imported from your bank account. Please be aware that CSV/OFX imported files of bank transactions do not support eReceipts.
+It's important to note that eReceipts are not generated for lodging expenses. Moreover, due to incomplete or inaccurate category information from certain banks, there may be instances of invalid eReceipts being generated for hotel purchases. If you choose to re-categorize expenses, a similar situation may arise. It's crucial to remember that our Expensify eReceipt Guarantee excludes coverage for hotel and motel expenses.
+
+# FAQ
+## What plan/subscription is required in order to manage corporate cards?
+Group Policy (Collect or Control plan only) 
```

Comment:
I think you may need a new line here but not 100% sure on that 
```suggestion
Group Policy (Collect or Control plan only) 

```

Reply:
![image](https://github.com/Expensify/App/assets/4073354/882257bd-c468-446f-993b-a38b39b351c0)

nevermind i was wrong here too